pkg/api/dryrunclient.go

```diff
@@ -95,11 +101,15 @@ func (d *DryRunClient) ContainerUnpause(ctx context.Context, container string) e
 }
 
 func (d *DryRunClient) CopyFromContainer(ctx context.Context, container, srcPath string) (io.ReadCloser, moby.ContainerPathStat, error) {
-	return nil, moby.ContainerPathStat{}, ErrNotImplemented
+	rc := io.NopCloser(strings.NewReader(""))
+	if _, err := d.ContainerStatPath(ctx, container, srcPath); err != nil {
+		return rc, moby.ContainerPathStat{}, fmt.Errorf(" %s Could not find the file %s in container %s", DRYRUN_PREFIX, srcPath, container)
+	}
+	return rc, moby.ContainerPathStat{}, nil
 }
 
 func (d *DryRunClient) CopyToContainer(ctx context.Context, container, path string, content io.Reader, options moby.CopyToContainerOptions) error {
-	return ErrNotImplemented
+	return nil
```

Comment:
To match the above, we might want to `ContainerStatPath(path.Dir(path))` and ensure it exists:

```
docker compose cp myfile svc:/this/path/does/not/exist
Error response from daemon: Could not find the file /this/path/does/not in container 281c3fe9c0f876d37c0953574e1fcb0385a133447fe39fb50a44369ef032b962
```

Reply:
I replicated the behaviour of the upstream `CopyToContainer` function which lets the docker daemon checking the existence of the target path, but I could definitely add this check for dry run mode